Train dense nn

ahijevyc opened the pull request:
TODO: fix random seed for reproducability

djgagne reviewed (commented):
I don't see any significant issues with the code as is, and your changes shouldn't interfere with what Charlie has been working on. My only suggestion would be to rename HWT_mode_train.py to train_mode_dnn.py to be consistent with train_mode_cnn.py.

ahijevyc commented:
Renamed training script train_mode.dnn.py to be consistent with train_mode_cnn.py.

Added function to hwtmode/data.py. It decomposes a feature into sine and cosine components.

Option to specify seed for random number generator.

Symbolic link to file in ahijevyc work directory for plotting reliability diagram and ROC curve.

djgagne commented:
@ahijevyc Can you pull the current master branch into your branch, fix conflicts, and push back to your branch? The commands should be
To create an upstream remote on your local git and pull from it:
`git remote add upstream git@github.com:NCAR/HWT_mode.git`
`git pull upstream master`
 You will then need to resolve conflicts by editing the conflicted files wherever there is a >>>>> or <<<<<<.
Then you can push the unconflicted changes to your github where they will be visible to the pull request.

ahijevyc commented:
DJG, I resolved the conflict and pushed to github.

On Sun, May 8, 2022 at 6:50 AM David Ahijevych ***@***.***> wrote:

> Hi DJG I’m away from computer till Wednesday but can do this soon after my
> return.
>
> Dave
>
>
> On May 6, 2022, at 4:42 PM, David John Gagne ***@***.***>
> wrote:
>
> ﻿
>
> @ahijevyc <https://github.com/ahijevyc> Can you pull the current master
> branch into your branch, fix conflicts, and push back to your branch? The
> commands should be
> To create an upstream remote on your local git and pull from it:
> git remote add upstream ***@***.***:NCAR/HWT_mode.git
> git pull upstream master
> You will then need to resolve conflicts by editing the conflicted files
> wherever there is a >>>>> or <<<<<<.
> Then you can push the unconflicted changes to your github where they will
> be visible to the pull request.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NCAR/HWT_mode/pull/23#issuecomment-1119993774>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEIPI2GNFOC6OZLDV4YTZ3DVIV7ZRANCNFSM5TBKGNGA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
>

djgagne commented:
Looks good to me. Merging. Thank you!
